README.md

```diff
@@ -86,13 +86,6 @@ source ~/.bashrc        # Most Linux Distros, some OSX
 source ~/.bash_profile  # Most OSX, some Linux Distros
 ```
 
-- Install the `nightly-2019-06-01` build of rust, `Rocket`, the web-framework relies on some advanced compiler options not yet included in the default build.
```

Comment:
Can we not delete this and instead have install instructions for 1.38.0, saying something like "hecate should work on stable versions of rust but is tested with x"

Reply:
Did you see the change above this? I added that note to the initial rust installation instructions.

Reply:
I did, but not everyone is going to know how to actually install it.

Reply:
We could include this in another line with those instructions?